.travis.yml

```diff
@@ -4,11 +4,14 @@ cache: bundler
 rvm:
   - 2.1.8
   - 2.2.4
-  - 2.3.0
+  - 2.3.1
 gemfile:
   - Gemfile
   - gemfiles/rails-4.2.gemfile
+  - gemfiles/rails-master.gemfile
 matrix:
   exclude:
     - gemfile: Gemfile
       rvm: 2.1.8
+    - gemfile: gemfiles/rails-master.gemfile
+      rvm: 2.1.8
```

Comment:
Any reason we're not excluding `gemfiles/rails-4.2.gemfile`? If we were, we could just drop 2.1.8.